tools/libressl/Makefile

```diff
@@ -8,9 +8,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libressl
-PKG_VERSION:=3.7.0
-PKG_HASH:=3fc1290f4007ec75f6e9acecbb25512630d1b9ab8c53ba79844e395868c3e006
-PKG_RELEASE:=1
```

Comment:
Any reason, why you decided to remove ``PKG_RELEASE``?

Reply:
It does not belong in tools correct

Reply:
So I made a new commit https://github.com/lovehackintosh/openwrt/commit/e5652422c34ebf932f212ce808edf5fa56957d7a ​
Could you review it?

Reply:
Looks good! I have just two small additions:
 -  I think you should drop the Fixes tag.
 - There is a typo in the commit subject, and my suggestion is that you are re-using the word "useless" so much.
    - You can use e.g.:``remove PKG_RELEASE as it is not used``, which makes the subject more clear that going through the Git history and you see only "useless", "useless", "useless" and for more details, you need to take a look at commit description. I hope you got the point which I tried to explain.

Reply:
https://github.com/lovehackintosh/openwrt/commit/511e0c682ba006796d62d7225a3fdcf33f026072

Reply:
It's better than it was, thanks!

Reply:
Let’s open a PR, thanks.